test/runtests.jl

```diff
@@ -760,6 +760,11 @@ end
         @test isapprox(1.0u"m", 1.1u"m"; rtol=0.2)
         @test !isapprox(1.0u"m", 1.1u"m"; rtol=0.05)
 
+        # Issue 465:
+        z = (1+im) * ones(2,3) * 1m
```

Comment:
```suggestion
        z = fill((1+im)m, 2, 3)
```
I think this would be clearer than the multiplication chain.

Reply:
agreed!